src/systems/waypoint-system.js

```diff
@@ -323,7 +329,24 @@ export class WaypointSystem {
       this.waitOneTick = false;
       return;
     }
+
+    const hashUpdated = window.location.hash !== "" && this.previousWaypointHash !== window.location.hash;
+
+    if (hashUpdated && this.initialSpawnHappened) {
+      this.previousWaypointHash = window.location.hash;
+      const spawnPointName = window.location.hash.replace("#", "");
+      const spawnPoint = this.ready.find(c => c.el.className === spawnPointName);
```

Comment:
`spawnPoint`/`spawnPointName` -> `waypoint`/`waypointName`

Reply:
I wonder whether there's any difference between `c.el.object3D.name` and `c.el.className`. I think the `object3D.name` matches what's in spoke and the gltf -- I didn't know/realize that the entity's `className` matched this as well. Not sure if that's always the case (e.g. if something else modifies an entity's `classList`)

Reply:
> `spawnPoint`/`spawnPointName` -> `waypoint`/`waypointName`

Renamed, makes more sense indeed, thanks!

> I wonder whether there's any difference between `c.el.object3D.name` and `c.el.className`. I think the `object3D.name` matches what's in spoke and the gltf -- I didn't know/realize that the entity's `className` matched this as well. Not sure if that's always the case (e.g. if something else modifies an entity's `classList`)

Yeah I was surprised also that it matches so I thought it was something specific from hubs. In any case I changed it to use `object3D.name` which makes feels more safe.